qds_sdk/engine.py

```diff
@@ -78,8 +78,16 @@ def set_engine_config(self,
                                  default_pool, enable_rubix)
         self.set_presto_settings(presto_version, custom_presto_config)
         self.set_spark_settings(spark_version, custom_spark_config)
-        self.set_airflow_settings(dbtap_id, fernet_key, overrides, airflow_version, airflow_python_version)
-        self.set_mlflow_settings(mlflow_version)
+        # because of same config dbtap_id used in mlflow and airflow,
+        # configs are getting created for both cluster type
+        if mlflow_version:
```

Comment:
can't we use a different var named `mlflow_dbtap_id ` for mlflow_version?

Reply:
done